cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -36,7 +38,7 @@
     "command": [
               "/bin/sh",
               "-c",
-              "/usr/local/bin/etcd --name etcd-{{ hostname }} --listen-peer-urls http://{{ hostname }}:{{ server_port }} --initial-advertise-peer-urls http://{{ hostname }}:{{ server_port }} --advertise-client-urls http://127.0.0.1:{{ port }} --listen-client-urls http://127.0.0.1:{{ port }} --data-dir /var/etcd/data{{ suffix }} --initial-cluster-state {{ cluster_state }} --initial-cluster {{ etcd_cluster }} 1>>/var/log/etcd{{ suffix }}.log 2>&1"
+              "if true; then /usr/local/bin/etcdctl migrate --data-dir=var/etcd/data{{ suffix }}; fi; /usr/local/bin/etcd --name etcd-{{ hostname }} --listen-peer-urls http://{{ hostname }}:{{ server_port }} --initial-advertise-peer-urls http://{{ hostname }}:{{ server_port }} --advertise-client-urls http://127.0.0.1:{{ port }} --listen-client-urls http://127.0.0.1:{{ port }} --data-dir /var/etcd/data{{ suffix }} --initial-cluster-state {{ cluster_state }} --initial-cluster {{ etcd_cluster }} 1>>/var/log/etcd{{ suffix }}.log 2>&1; if true; then sleep 10; /usr/local/bin/etcdctl rm /registry --recursive; fi"
```

Comment:
So what's the procedure for rollback? Is it possible to rollback using "etcdctl migrate" (with different flags)?
@hongchaodeng @xiang90 ^^

I think that it should be possible to do rollback with this approach - what we should do is to pass set some env variable to desired storage format and based on that choose what migration we should do (even if it's not possible to do it with etcdctl migrate).

Or to clarify - my idea for rollback would be:
- rollback just storage format first (with the same manifest, but setting different env variable)
- once we're back to etcd2 API, rollback to the previous release
WDYT?